### ISSUE TITLE
[Suggestion] Screenshots

### ISSUE THREAD

**Issue by frakman1** [flask-admin/flask-admin#1933]
This project's README and documentation page could use some screenshots. Lots of them.

**Comment by hasansezertasan** [comment]
I'll do you one better, an online demo.

Here are some projects that have online demos: Flask App Builder, Starlette Admin, SQLAdmin, and FastAPI Admin.

WDYT? @pallets-eco/flask-admin

**Comment by samuelhwilliams** [comment]
Yeah this feels like a good idea - a bit of a review/refresh might be nice.

**Comment by ElLorans** [comment]
addressed in https://github.com/pallets-eco/flask-admin/commit/821eb7ac0361eea6577a82a8c4b939cf5587439f , but happy to receive PRs adding more screenshots
